src/client/angle/angle-bridge-data.ts

```diff
@@ -0,0 +1,121 @@
+import { EthAddress } from "@aztec/barretenberg/address";
+import { EthereumProvider } from "@aztec/barretenberg/blockchain";
+import { Web3Provider } from "@ethersproject/providers";
+import { IStableMaster, IStableMaster__factory } from "../../../typechain-types";
+import { createWeb3Provider } from "../aztec/provider";
+import {
+  AssetValue,
+  AuxDataConfig,
+  AztecAsset,
+  AztecAssetType,
+  BridgeDataFieldGetters,
+  SolidityType,
+} from "../bridge-data";
+
+export class AngleBridgeData implements BridgeDataFieldGetters {
```

Comment:
Why is there no function for APR etc? Seems like an important metric for a yield bearing asset.